Backend/DataAccessLayer/DataAccessLayer.csproj

```diff
@@ -75,6 +75,10 @@
     <Compile Include="Migrations\201903111003125_upate-sessions-userNotUnique.Designer.cs">
       <DependentUpon>201903111003125_upate-sessions-userNotUnique.cs</DependentUpon>
     </Compile>
+    <Compile Include="Migrations\201903122358472_pending-migration.cs" />
```

Comment:
why is there a migration being added? I thought this was just VueX

Reply:
created a pull request for the front end